src/core/qgsclipper.h

```diff
@@ -402,6 +533,8 @@ inline QgsPointXY QgsClipper::intersect( const double x1, const double y1,
       r_n = ( y1 - MIN_Y ) * ( MAX_X - MIN_X );
       r_d = -( y2 - y1 )   * ( MAX_X - MIN_X );
       break;
+    default:
```

Comment:
use explicit case instead of default here -- it will help with future work as warnings are raised for unhandled enum values.

Reply:
ok!